features/installer/install_command.feature

```diff
@@ -0,0 +1,45 @@
+@cli
+Feature: Sylius Install Feature
+  In order to install Sylius via CLI
+  As an Administrator
+  I want to run an installation command
+
+  Scenario: Running install setup command
+    When I run a command "sylius:install:setup"
+    Then I should see output "Please enter a currency code (For example "GBP") or press ENTER to use "USD"."
+    And I should see output "In which currency can your customers buy goods?"
+    
+  Scenario: Choosing default currency
+    Given I do not provide a currency
+    When I run a command "sylius:install:setup"
```

Comment:
Btw. we are setting up only that command inside our contexts, what do you think about renaming this step to "I run Sylius CLI installer"?